sys/shell/commands/sc_gnrc_icmpv6_echo.c

```diff
@@ -322,7 +322,8 @@ static void _pinger(_ping_data_t *data)
 }
 
 static void _print_reply(_ping_data_t *data, gnrc_pktsnip_t *icmpv6,
-                         ipv6_addr_t *from, unsigned hoplimit)
+                         ipv6_addr_t *from, unsigned hoplimit,
+                         gnrc_netif_hdr_t* netif_hdr)
```

Comment:
I think it might be a more consistent API to expect `int16_t rssi` directly here, not the full netif header. But I'm not insisting on this.

Reply:
I liked the implicit cast, but I don't have strong feelings either way. Changed it to pass `rssi` directly.